modules/installation/nav-installation-guide.adoc

```diff
@@ -15,14 +20,24 @@ ifeval::[{suma-content} == true]
 *** xref:install-server-unified.adoc[Install Server]
 *** xref:install-proxy-unified.adoc[Install Proxy]
```

Comment:
If I am not wrong, this is what generates:
![image](https://user-images.githubusercontent.com/4226070/69711677-bf479380-1101-11ea-8aaf-24524f4fe26b.png)
The procedure is incorrect. The user must NOT mix Uyuni Server and SUSE Manager Proxy. Uyuni Proxy is to be used, and therefore SLE15SP1 channels are not required, rather than Leap 15.1 and the proxy channels for Uyuni.

Reply:
tracked as https://github.com/SUSE/doc-susemanager/issues/904

Reply:
You did not add #904 to the epic #903. Just did it now.